docs/layers/primitive-polygon-layer.md

```diff
@@ -1,99 +0,0 @@
-<p class="badges">
```

Comment:
I think we should have docs for layers even if not exposed. Can we move it into the layer as a README.md?